<launch>

<node pkg="atlas_drone" type="drone_ai" name="drone_ai" output="screen">
    <rosparam file="$(find atlas_drone)/config/navigation.yaml" command="load" />
</node>

</launch>
